moto/ivs/models.py

```diff
@@ -0,0 +1,134 @@
+"""IVSBackend class with methods for supported APIs."""
+from typing import Optional, Any
+from moto.core import BaseBackend, BackendDict
+from moto.ivs.exceptions import ResourceNotFoundException
+from moto.utilities.paginator import paginate
+from moto.moto_api._internal import mock_random
+
+
+class IVSBackend(BaseBackend):
+    """Implementation of IVS APIs."""
+
+    PAGINATION_MODEL = {
+        "list_channels": {
+            "input_token": "next_token",
+            "limit_key": "max_results",
+            "limit_default": 100,
+            "unique_attribute": "arn",
+        },
+    }
+
+    def __init__(self, region_name: str, account_id: str):
+        super().__init__(region_name, account_id)
+        self.channels: list[dict[str, Any]] = []
+
+    def create_channel(
+        self,
+        authorized: bool,
+        insecure_ingest: bool,
+        latency_mode: str,
+        name: str,
+        preset: str,
+        recording_configuration_arn: str,
+        tags: dict[str, str],
```

Comment:
```suggestion
        tags: Dict[str, str],
```
If I understand correctly, Python <= 3.8 doesn't support lowercase dict-types like this - it throws an error that `TypeError: 'type' object is not subscriptable`.
That means importing `typing.Dict` ofcourse, and changing this for all instances.

Failing tests:
https://github.com/getmoto/moto/actions/runs/6458787497/job/17534778481?pr=6894

Reply:
Thanks, that's an easy fix!